basics/src/test/java/basics/AmpegTest.java

```diff
@@ -0,0 +1,175 @@
+	
+	
+	@Before
+	public void PreinitChorme() {	
+		
+		ChromeDriverPath = "C:\\Users\\techo\\Desktop\\Selenium\\chromedriver.exe";
+		
+		System.setProperty(WEB_DRIVER_SYSTEM_PROPERTY, ChromeDriverPath);
+		driver = new ChromeDriver();
+		ChromeDriverDocPage = new ChromeDriverDocPage(driver);
+	}
+	
+	@Test
+	public void CanDisplayHomePageTest() {
+		ChromeDriverHomePage HomePage = ChromeDriverDocPage.DisplayHomePage();
+		String currentURL = driver.getCurrentUrl();
+		String HomeUrl = HomePage.getHomeUrl();
+		assertEquals("cant not go to url: " + HomeUrl, currentURL, HomeUrl);
+		
+		driver.quit();
+	}
+	
+	@Test
+	public void CanDisplayProductPageTest() {
+		String ProductPageUrl = "https://ampeg.com/products/";
+		
+		ChromeDriverProductPage ProductPage = ChromeDriverDocPage
+				.DisplayHomePage()
+				.DisplayProductPage();
+		String currentUrl = driver.getCurrentUrl();
+		
+		assertEquals("cant not go to url: " + ProductPageUrl, currentUrl, ProductPageUrl);
+		
+		driver.quit();
+	}
+	
+	@Test
+	public void CanClickToViewProductSeriesTest1() {
+		String Series = "pro";
+		String alt = "SVT Pro Series: Bass heads";
+		String ProductSeriesUrl = "https://ampeg.com/products/pro/";
+		
+		ChromeDriverProductSeries ProductSeriesPage = ChromeDriverDocPage
+				.DisplayHomePage()
+				.DisplayProductPage()
+				.DisplayProductSeries(alt, Series);
+		String currentUrl = driver.getCurrentUrl();
+		
+		assertEquals("cant not go to url: " + ProductSeriesUrl, currentUrl, ProductSeriesUrl);
+		driver.quit();
+	}
+	
+	@Test 
+	public void CanClickSVT_3PRO(){
+		String Series = "pro";
+		String SVT_3PRO_Path = "//img[@alt = 'SVT-3PRO']/..";
+		String SVT_3PRO_URL = "https://ampeg.com/products/pro/svt3pro/";
+		String alt = "SVT Pro Series: Bass heads";
+		
+		ChromeDriverProductDetail ProductDetailPage = ChromeDriverDocPage
+				.DisplayHomePage()
+				.DisplayProductPage()
+				.DisplayProductSeries(alt, Series)
+				.DisplayProductDetail(SVT_3PRO_Path, SVT_3PRO_URL);
+		
+		String currentUrl = driver.getCurrentUrl();
+		
+		assertEquals("cant not go to url: " + SVT_3PRO_URL, currentUrl, SVT_3PRO_URL);
+		
+		driver.quit();
+	}
+	
+	@Test
+	public void CanClickBreadcrumb_Home() {
+		String Series = "pro";
+		String SVT_3PRO_Path = "//img[@alt = 'SVT-3PRO']/..";
+		String SVT_3PRO_URL = "https://ampeg.com/products/pro/svt3pro/";
+		String alt = "SVT Pro Series: Bass heads";
+		
+		ChromeDriverHomePage Homepage = ChromeDriverDocPage
+				.DisplayHomePage()
+				.DisplayProductPage()
+				.DisplayProductSeries(alt, Series)
+				.DisplayProductDetail(SVT_3PRO_Path, SVT_3PRO_URL)
+				.BackToHome();
+		
+		String currentUrl = driver.getCurrentUrl();
+		
+		assertEquals("cant not go to url: " + Homepage.getHomeUrl(), currentUrl, Homepage.getHomeUrl());
+		driver.quit();
+	}
+	
+	@Test
+	public void CanClickToViewProductSeriesTest2() {
+		String Series = "classic";
+		String alt = "Classic Series: Bass heads & enclosures";
```

Comment:
Locators should be in the PageObject since they are a location strategy for things on the web page.